versions/appinfo/0.0.3.toml

```diff
@@ -0,0 +1,31 @@
+[metadata]
+configVersion = "0.0.3"
+
+[environments]
+bananas = { }
+
+[environments.default.android]
+platform = "Android"
+available = true
+minimumVersion = "1.0.0"
+recommendedVersion = "1.2.0"
+releaseFlags = {
+    featureOne = true
+    featureTwo = false
+}
+
+[environments.default.ios]
+platform = "iOS"
+available = true
+minimumVersion = "2.0.1"
+recommendedVersion = "2.2.0"
+someOtherProperty = "fish chips beans"
```

Comment:
The error you're getting is very strange 🤷 

The only (real) difference between the YAML and TOML is that `someOtherProperty` is above `releaseFlags` in the TOML and not below as in the YAML. I can't imagine why this would cause an issue though. 

One other thing I noticed, in TOML alphabetical ordering of dotted keys is recommended - but not enforced. I guess it's possible that the parser you're using could be adhering to that recommendation overzealously. But the JSON looks fine to me. 

Either way, It feels more like a parser issue than an issue in your code. You could try another parser to see if that clears it 🤔